config/locales/diaspora/en.yml

```diff
@@ -683,6 +683,7 @@ en:
         subject: "%{name} has mentioned you on diaspora*"
         limited_post: "You were mentioned in a limited post."
     private_message:
+        subject: "Diaspora: Private Message"
```

Comment:
@svbergerem 
I too thought of the same but if we check the mail message we are getting it is

"**Subject**- _There’s a new private message in diaspora* for you to check out_" 
 which shows the body in italic and if we add the content it would be confusing for User

PFA 

![test](https://cloud.githubusercontent.com/assets/5456598/16690753/095eeea8-4547-11e6-98a7-a8a7ce4e9497.png)